spec/spec.md

```diff
@@ -1477,6 +1527,35 @@ The following JSON Schema Draft 7 definition summarizes the rules above:
 
 </tab-panels>
 
+## Credential Format Designations
+
+Within the _Presentation Exchange_ specification, there are numerous sections 
+where Verifiers and Holders convey what credential variants they support and 
+are submitting. The following are the normalized references used within the 
+specification:
+
+- `jwt` - the format is a [JSON Web Token](https://tools.ietf.org/html/rfc7797) 
+  that will be submitted in the form of a JWT encoded string. Expression of 
+  supported algorithms in relation to this format ****MUST**** be conveyed using an `alg` property 
+  paired with values that are identifiers from the 
+  [RFC 7518 JSON Web Algorithms](https://tools.ietf.org/html/rfc7518) registry.
+- `jwt-vc`, `jwt-vp` - these formats are [JSON Web Tokens](https://tools.ietf.org/html/rfc7797) 
+  that will be submitted in the form of a JWT encoded string, and the body of the decoded 
+  JWT string is defined in the [JSON Web Token section](https://www.w3.org/TR/vc-data-model/#json-web-token) 
+  of the W3C Verifiable Credentials specification. Expression of supported algorithms in 
+  relation to these formats ****MUST**** be conveyed using an `alg` property paired with values that 
+  are identifiers from the [RFC 7518 JSON Web Algorithms](https://tools.ietf.org/html/rfc7518) registry. 
+- `vc`, `vp` - these formats are [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) 
+  that will be submitted in the form of a JSON object. Expression of supported 
+  algorithms in relation to these formats ****MUST**** be conveyed using a `type` property 
+  paired with values that are identifiers from the 
+  [Linked Data Cryptographic Suite Registry](https://w3c-ccg.github.io/ld-cryptosuite-registry/).
+- `ldp` - this format is defined in the [W3C CCG Linked Data Proofs](https://w3c-ccg.github.io/ld-proofs/) 
```

Comment:
`ldp` does not exist outside of JSONLD (that I know of), and we've already established `vc` and `vp` covering JSONLD formats... so this is redundant

Reply:
Are we allowing for some sort of "detached" linked data proofs?

Reply:
`ldp` are used with capabilities, they are the Linked Data version of a vanilla JWT.... if we are gonna allow vanilla JWT, we might as well allow ldp... here is an ldp that is not a VC...

https://github.com/transmute-industries/vc.js/blob/master/packages/linked-data-proof/src/__fixtures__/expected/doc-signed.json

Reply:
Ok... they are still used in JSONLD documents but this format is meant for documents other than VCs and VPs...

Reply:
then we should remove "JWT"... that was my initial point.